components/sankey/elements/tooltip.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Tooltip
+page_title: Sankey Tooltip
+description: Tooltip of the Sankey Diagram for Blazor.
+slug: sankey-tooltip
+tags: telerik,blazor,sankey,diagram,chart,tooltip
+published: True
+position: 9
+---
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+# Sankey Tooltip
```

Comment:
Perhaps the content will be more easier to understand if the documentation mentions "link tooltip" and "node tooltip" more often, including in the headings - for example, Node Tooltip instead of NodeTemplate.

The parent configuration tag is one, but the tooltips are two and are not directly related.